frontend/src/static/css/v_authorship.scss

```diff
@@ -136,6 +111,18 @@
         margin-left: .5rem;
         text-decoration: none;
       }
+
+      .icons {
+        margin-right: 8px;
+        vertical-align: middle;
+      }
+
+      .fileTypeLabel {
+        border-radius: 5px;
+        display: inline-block;
+        font-size: 15px;
```

Comment:
is it possible to standardise the size across all file type labels? i.e. using our custom `medium-font` css